README.md

```diff
@@ -46,12 +46,15 @@ conda activate transxpress
 conda config --add channels bioconda
 conda config --add channels conda-forge
 conda config --set channel_priority false
-conda install snakemake fastqc trimmomatic trinity spades transdecoder biopython samtools bowtie2 infernal hmmer kallisto blast r bioconductor-edger seqkit wget r-tidyverse python=3.6 numpy=1.14 scipy=1.0 theano=1.0.1 six==1.11 parallel
+conda install "snakemake>=5" fastqc transdecoder samtools infernal hmmer kallisto blast=2.10 seqkit wget
+conda install r bioconductor-edger r-tidyverse 
+conda install trinity=2.11 trimmomatic bowtie2 python=3.6 biopython numpy=1.16 scipy=1.0 theano=1.0.1 six==1.11 parallel spades
```

Comment:
Not sure if it's a good strategy to fix the trinity version (2.11) here. That means we need to update the README with each new release of trinity. Is in necessary?